plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/web/utils/WebExpectedConditionsUtils.java

```diff
@@ -140,43 +153,7 @@ public static String findNewWindowHandle(Set<String> existingHandles, int timeou
      * @throws ru.sbtqa.tag.pagefactory.exceptions.WaitException TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)